CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## Version 1.0.11
```

Comment:
@littleaj as we changed the public facing method do we need to provide description of what has been changed in the implementation of the method? Rest looks fine.

Reply:
I can modify this to reflect that this fixes a regression in trackTrace/trackException